header.php

```diff
@@ -23,6 +23,31 @@
 	<!-- wordpress head functions -->
 	<?php wp_head(); ?>
 
+  <link rel="apple-touch-icon" sizes="180x180" href="<?php echo get_stylesheet_directory_uri() ?>/library/icons/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="<?php echo get_stylesheet_directory_uri() ?>/library/icons/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="<?php echo get_stylesheet_directory_uri() ?>/library/icons/favicon-16x16.png">
+  <link rel="manifest" href="<?php echo get_stylesheet_directory_uri() ?>/library/icons/site.webmanifest">
+  <link rel="mask-icon" href="<?php echo get_stylesheet_directory_uri() ?>/library/icons/safari-pinned-tab.svg" color="#4881cb">
+  <link rel="shortcut icon" href="<?php echo get_stylesheet_directory_uri() ?>/library/icons/favicon.ico">
+  <meta name="apple-mobile-web-app-title" content="PPSri.org">
+  <meta name="application-name" content="PPSri.org">
+  <meta name="msapplication-TileColor" content="#4881cb">
+  <meta name="msapplication-TileImage" content="<?php echo get_stylesheet_directory_uri() ?>/library/icons/mstile-150x150.png">
+  <meta name="msapplication-TileImage" content="<?php echo get_stylesheet_directory_uri() ?>/library/icons/mstile-144x144.png">
+  <meta name="msapplication-config" content="<?php echo get_stylesheet_directory_uri() ?>/library/icons/browserconfig.xml">
+  <meta name="theme-color" content="#d1dcf2">
+
+<?php if ( ! is_user_logged_in() ) { ?>
+  <!-- Global site tag (gtag.js) - Google Analytics -->
```

Comment:
trackers make me sad : ( but I understand clients want them.

Reply:
For the record, the only tracker they use is the standard Analytics, but GA wants you to use their global site tag now as a way of managing it. I suppose I could go around it and only install the standard GA… but I dont know if PPS will want to use another service like AdWords in the future.